products/bellsoft-liberica.md

```diff
@@ -46,50 +46,50 @@ auto:
 # Extended support dates can be found on https://bell-sw.com/roadmap/.
 releases:
 -   releaseCycle: "20"
-    releaseDate: 2023-03-22
```

Comment:
20+37 has been released on 2023-03-22 (see https://github.com/endoflife-date/release-data/blob/74489af2880e199b375cdb570e5c0663d2996421/releases/bellsoft-liberica.json#LL51C13-L51C23) but looks like the `latest.py` script does not consider this versions properly.

Reply:
Fixed by #3016.